code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -909,16 +915,33 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 			update_icons()
 		return
 
-/mob/living/silicon/robot/verb/unlock_own_cover()
+/mob/living/silicon/robot/verb/toggle_own_cover()
 	set category = "Robot Commands"
-	set name = "Unlock Cover"
-	set desc = "Unlocks your own cover if it is locked. You can not lock it again. A human will have to lock it for you."
-	if(locked)
-		switch(alert("You can not lock your cover again, are you sure?\n      (You can still ask for a human to lock it)", "Unlock Own Cover", "Yes", "No"))
-			if("Yes")
-				locked = 0
-				update_icons()
-				to_chat(usr, "You unlock your cover.")
+	set name = "Toggle Cover"
```

Comment:
Still shows as Toggle Cover to normal borgs.

Reply:
I tried making it a seperate verb for ERT borgs, but I couldn't figure out a good way to do it, since it would show up for normal borgs too.

Also tried making the name a variable, but that doesn't work since all the `set` stuff has to be stated first.